src/Shared/SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure/Data/Entities/HistoricalPaymentEntity.cs

```diff
@@ -1,25 +0,0 @@
-﻿using System.ComponentModel.DataAnnotations;
-
-namespace SFA.DAS.ProviderPayments.Calc.Shared.Infrastructure.Data.Entities
-{
-    public class HistoricalPaymentEntity : PaymentEntity
```

Comment:
What happened to `HistoricalPaymentsEntity`? was this a duplicate?

Reply:
yes, you added it in the local refunds repo and Carl added it in the shared repo